src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,25 @@
+{
+	"schemaVersion": 1,
+	"id": "bismuthserver",
+	"version": "${version}",
+	"name": "Bismuth Server",
+	"description": "Custom server for Bismuth's needs.",
+	"authors": [
+		"Bismuth!"
+	],
+	"contact": {
+		"homepage": "https://github.com/BismuthServer",
+		"sources": "https://github.com/BismuthServer/BismuthServer"
+	},
+	"environment": "*",
+	"mixins": [
+		"mixins.bismuthserver.json",
+		"mixins.bismuthserver.movablete.json",
+		"mixins.bismuthserver.scoreboard.json",
+		"mixins.bismuthserver.ebictranslononation.json"
```

Comment:
Comment ebictranslononation mixins out, they weren't used previously. I should at some point remove the code completely, it was just an experiment from Nessie